src/main/java/webserver/HttpRequest.java

```diff
@@ -0,0 +1,60 @@
+package webserver;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Map;
+
+import static util.HttpParser.*;
+
+public class HttpRequest {
+    private final String HTTP_METHOD;
```

Comment:
대문자 표기법은 상수에 사용됩니다.  final을 사용했도 카멜 표기법으로 바꿔 주세요.
```suggestion
    private final String method;
```